src/Sylius/Bundle/AttributeBundle/Form/EventListener/BuildAttributeValueFormListener.php

```diff
@@ -11,47 +11,42 @@
 
 namespace Sylius\Bundle\AttributeBundle\Form\EventListener;
 
-use Sylius\Component\Attribute\Model\AttributeTypes;
-use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Sylius\Component\Registry\ServiceRegistryInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\Form\FormEvent;
 use Symfony\Component\Form\FormEvents;
 use Symfony\Component\Form\FormFactoryInterface;
 
 /**
- * Form event listener that builds product property form dynamically based on product data.
- *
- * @author Saša Stamenković <umpirsky@gmail.com>
- * @author Leszek Prabucki <leszek.prabucki@gmail.com>
- * @author Paweł Jędrzejewski <pawel@sylius.org>
- * @author Liverbool <liverbool@gmail.com>
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
  */
 class BuildAttributeValueFormListener implements EventSubscriberInterface
 {
     /**
-     * Form factory.
-     *
      * @var FormFactoryInterface
      */
     protected $factory;
 
     /**
-     * Attributes subject name.
-     *
+     * @var ServiceRegistryInterface
+     */
+    protected $attributeTypeRegistry;
+
+    /**
      * @var string
      */
     protected $subjectName;
 
     /**
-     * Constructor.
-     *
-     * @param FormFactoryInterface $factory
-     * @param string               $subjectName
+     * @param FormFactoryInterface     $factory
+     * @param ServiceRegistryInterface $attributeTypeRegistry
+     * @param string                   $subjectName
      */
-    public function __construct(FormFactoryInterface $factory, $subjectName)
+    public function __construct(FormFactoryInterface $factory, ServiceRegistryInterface $attributeTypeRegistry, $subjectName)
     {
         $this->factory = $factory;
         $this->subjectName = $subjectName;
+        $this->attributeTypeRegistry = $attributeTypeRegistry;
```

Comment:
Could you move it one line higher? So the assignments would be in the same order as parameters :)